src/cdk/constructs/config/make-default-config.ts

```diff
@@ -13,6 +13,9 @@ function createDefaultJSON (config: object): void {
       fs.mkdirSync(configDirPath)
     }
     const json: string = JSON.stringify(config, null, 2)
+
+    JsonValidation.validateFileJson(JsonValidation.getDefaultConfigSchema(), json)
```

Comment:
it would be great if you could hook up this validation, before every deploy/deployCi function from boostrap.ts class, so the deploy will fail-fast in case of wrong config file.